src/app/components/home/home.component.ts

```diff
@@ -31,10 +36,24 @@ export class HomeComponent {
         return this._mapToStoresWithTagsQueryModel(stores, tags);
       })
     );
+  readonly productsList$: Observable<CategoryWithProductsQueryModel[]> =
```

Comment:
it should be 2 separate observables, one for category id 5 and another for 2

Reply:
Fixed